tests/text/EnglishPluralizerTest.php

```diff
@@ -119,9 +119,9 @@ public function testSingularForm($output, $input) {
 
 	/**
 	 * @dataProvider providerForWrongType
-	 * @expectedException \InvalidArgumentException
 	 */
 	public function testWrongTypeToSingularizeThrowsException($wrong) {
+		$this->expectException(\InvalidArgumentException::class);
```

Comment:
I think we can add the `expectExceptionMessage` to assert the expected exception message is same as result.

Reply:
Done